src/styles/blocks/product.scss

```diff
@@ -0,0 +1,155 @@
+/* Products */
+
+.catalog {
+  --column-count: 1;
+
+  display: grid;
+  grid-auto-flow: row;
+  justify-content: center;
+  justify-items: center;
+
+  grid-template-rows: 2fr;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: $horizontal-card-gap $vertical-card-gap;
+  padding: 50px 0;
+  box-sizing: border-box;
+  margin: auto;
+}
+
+/* #region Breakpoints */
+
+@media (min-width: 488px) {
+  .catalog {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    --column-count: 4;
+  }
+}
+
+/* #endregion */
+
+.product-card {
+  box-sizing: border-box;
+  width: $product-card-width;
+
+  // border: 1px solid $card-border-color;
```

Comment:
It's usually a good idea to remove commented-out code to keep the codebase clean. If you need to remember what you've done in the past, you can always use version control systems like Git. This way, your code will be cleaner and easier to read.